packages/multichain-network-controller/src/MultichainNetworkController.ts

```diff
@@ -47,6 +48,11 @@ export class MultichainNetworkController extends BaseController<
       state: {
         ...getDefaultMultichainNetworkControllerState(),
         ...state,
+        multichainNetworkConfigurationsByChainId: {
+          // We can keep the current network as a hardcoded value
+          // since it is not expected to add/remove networks yet.
+          ...AVAILABLE_MULTICHAIN_NETWORK_CONFIGURATIONS,
+        },
```

Comment:
```suggestion
         // We can keep the current network as a hardcoded value
         // since it is not expected to add/remove networks yet.
        multichainNetworkConfigurationsByChainId: AVAILABLE_MULTICHAIN_NETWORK_CONFIGURATIONS,

Reply:
Ah sorry @cryptodev-2s, looks like it got merged before your comment. I'll open a new PR for this right away

Reply:
done in https://github.com/MetaMask/core/pull/5642/files